apps/ideas/src/components/modules/WorkshoppingPage/like-button-test.js

```diff
@@ -0,0 +1,17 @@
+// test file for the like button on the workshopping page
+// source: https://stackoverflow.com/a/53685236
+
+import React, { useState, useEffect } from "react";
+
+const HeartButton = () => {
```

Comment:
Can we make sure the name of this file is the same as the component's name? E.g. rename the file "HeartButton.js" instead of "like-button-test.js"

Reply:
new file looks good for the naming... but can we make sure this one gets deleted to avoid having duplicate components?